modules/introduction/partials/new_features-76_4.adoc

```diff
@@ -33,17 +33,14 @@ curl --get -u <username:password> \
 * The Search Service now supports the cosine similarity metric for xref:vector-search:vector-search.adoc[Vector Search indexes].
 For more information about Vector Search similarity metrics, see xref:search:child-field-options-reference.adoc[Child Field Options].
 
+* The Search Service now supports a new option for optimizing Vector Search indexes: *memory-efficient*.
+Choose this option to prioritize reduced memory and resource usage for Vector Searches, at the cost of accuracy and latency. 
+For more information, see xref:search:child-field-options-reference.adoc[Child Field Options].
+
 * The Search Service has added a new object to JSON Search queries. 
 Use this new object to view detailed query debugging information and resolve query errors in the Web Console or through the REST API.
 For more information about how to run a query with this new object, see xref:search:search-request-params.adoc#validate[the validate property], xref:search:simple-search-rest-api.adoc#example-validate-a-search-query[Run a Simple Search with the REST API and curl/HTTP] or xref:search:simple-search-ui.adoc#example-validate-a-search-query[Run A Simple Search with the Web Console].
 
-[#new-features-764-eventing-service]
```

Comment:
Can you restore this section please? Thanks

Reply:
Ah, must have been a bad merge, one sec